### PR TITLE
add ability to include the number of posts fetched in the subject and add ability to skip sending if no posts fetched

### PR DESCRIPTION
**Pull request by arranf**

### CODE REVIEW

**Comment by AntoniosBarotsis** [comment]
I always had so many rss feeds that I never thought of adding this xd

I added a short message for when `SKIP_IF_NO_NEW_POSTS` is not set but there were no posts found to make it more obvious that the empty HTML is not an internal bug.

File: README.md [hunk 1]
```diff
@@ -47,14 +47,15 @@ shell or as entries in a `.env` file.
 
 - `EMAIL_ADDRESS`: the mail address you will use to send the emails
 - `RECIPIENT_ADDRESSES`: comma delimited list of recipient email addresses
-- `SUBJECT`: the email subject (could be `rss2email`)
+- `SUBJECT`: the email subject (could be `rss2email`). `$POST_COUNT` is replaced with the number of posts fetched.
```
Comment (AntoniosBarotsis) [inline README.md:50]: Not a huge fan of this, could you make post count a togglable flag like everything else?

**Review by AntoniosBarotsis — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (arranf) [reply README.md:50]: My thinking here is that that would not work with custom subjects in any way

If I wanted to change the subject to `Posts from Science Feeds: 12 Posts` I wouldn't be able to unless we had substitution variables.

Comment (AntoniosBarotsis) [reply README.md:50]: Wouldn't it work to have a flag that appends the number of posts to the subject? So you can set your subject to be "Science feeds" and if you enable the post count you get something like "Science feeds (42 posts)" or did I misunderstand something

Comment (AntoniosBarotsis) [reply README.md:50]: I can't really think of any other way myself so its fine for now :)

**Pull request merged**